README.md

```diff
@@ -6,9 +6,35 @@ Codidact is community-driven Q&A software: created, maintained, used, and run by
 money-making company behind this.
 
 We're currently in the early stages of development, but once we have a working MVP, we'll be hosting a public Codidact instance
-for everyone to use. The software will also be available to download and run yourself to host your own instance under your own
+for everyone to use. The software will also be available to download and run yourself to host your instance under your own
 rules.
 
+## Installation
+These instructions are for setting up and running a local instance of Codidact.
```

Comment:
Or alternatively add a "development" before instance here and later a link to a production guide.

Reply:
I think I'll go this route for now. I can imagine a production setup document being its own file in the future.